.github/workflows/build.yml

```diff
@@ -0,0 +1,139 @@
+name: Test
+on:
+  push:
+
+jobs:
+  lint-docs:
+    runs-on: macos-11
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14.19.3'
```

Comment:
16?

Reply:
Yeah looks like that's sitting in `0.5.0-stage` waiting for merge. I went ahead and updated it here anyway as well.